doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -307,8 +309,58 @@ Most cloud storage architectures incorporate a number of clustered storage nodes
 
 <a name="3.5"></a>
 ## 3.5. Cloud Topology
-Content to be developed along the following lines
-*	How do we define the different deployment types?
-    - Edge
-    -	Core DC 
-    -	etc.
+
+Please note: This section focuses on central DC based cloud deployments, Edge is currently out of scope.
+
+In order to provide the expected availability for any given service, a number of different OpenStack deployment topologies can be considered.  This section explores the main options and highlights the characteristics of each.  Ultimately the decision rests with the operator to achieve specific availability target taking into account use case, data centre capabilities, economics and risks.
+
+Availability of any single OpenStack cloud is dependent on a number of factors including: 
+-	environmental – dual connected power and PDUs, redundant cooling, rack distribution etc. 
+-	resilient network fabric – ToR (leaf), spine, overlay networking, underlay networking etc.   It is assumed that all network components are designed to be fault tolerant and all OpenStack controllers, computes and storage are dual-homed to alternate leaf switches.
+-	controller nodes setup in-line with the vendor recommendation (e.g. min 3 physical nodes)
+-	network nodes (where applicable) 
+-	storage nodes setup in line with the vendor recommendation
+-	compute nodes – enough to handle the workload after local failure scenario
+
```

Comment:
@iangardner22 do we need to include some of the Topology Considerations deleted from Chapter 4?

Reply:
OK - what of the following do you think we should keep or merge with the new text?

> A Telco cloud will be deployed in multiple locations (“sites”) of varying size and capabilities (HVAC, for example); or stated slightly differently, multiple telco clouds (i.e. OpenStack end points) will be deployed and they all contain isolated resources that do not rely on each other.   The application must span such end points in order to provide the required service SLA Irrespective of the nature of the deployment characteristics (number of racks, number of hosts, etc.), the intent of the architecture would be to allow VNFs to be deployed in these sites as needed without major changes; if not all as many as possible.
> -	Large data center capable of hosting thousands of servers and the networking to support them
> -	Mini data center (such as a central office) capable of hosting up to a hundred servers
> -	Edge (not customer premise) capable of hosting between ten to fifty servers
> 
> Host profiles (SW Host profile + HW host profile) “partition” the cloud into pseudo sub-clouds, for example, hosts targeted for basic instance types, network intensive instance types and compute intensive instance types. This can happen, because of specific hardware adds and/or hardware and software configurations.  Depending upon the workload types and the capacity requirements, cloud providers and operators may choose to support the instance types with targeted hardware (different number of sockets, RAM, clock speeds, etc.) and host profiles or choose common hardware and minimize the number of host profiles (for example, network intensive and compute intensive types using a common host profile).
> 
> As we get away from the large data centers to the smaller sites it becomes progressively difficult to be able to create enough capacity for each of these instance types in support of their target VNFs or to have a mix of hardware targeted for each instance type.

Reply:
@iangardner22 
I would try and keep:
A Telco cloud will be deployed in multiple locations (“sites”) of varying size and capabilities (HVAC, for example); or stated slightly differently, multiple telco clouds (i.e. OpenStack end points) will be deployed and they all contain isolated resources that do not rely on each other. The application must span such end points in order to provide the required service SLA Irrespective of the nature of the deployment characteristics (number of racks, number of hosts, etc.), the intent of the architecture would be to allow VNFs to be deployed in these sites as needed without major changes; if not all as many as possible.

    Large data center capable of hosting thousands of servers and the networking to support them
    Mini data center (such as a central office) capable of hosting up to a hundred servers
    Edge (not customer premise) capable of hosting between ten to fifty servers
++++++++++++

The other piece about Host profiles should probably go to the relevant section.

Reply:
OK I will ensure that is incorporated thank you.

The host profiles piece - where would you suggest this goes?

Reply:
@pgoyal01 - I am incorporating your suggestion but what about host aggregates - where would you recommend?